docs/rest_api/reports.rst

```diff
@@ -0,0 +1,311 @@
+Reports
+#######
+
+Use this endpoint to obtain details on Mautic's Reports.
+
+**Using Mautic's API Library**
+
+You can interact with this API through the :xref:`Mautic API Library` as follows, or use the various http endpoints as described in this document.
+
+.. code-block:: php
+
+   <?php
+   use Mautic\MauticApi;
+   use Mautic\Auth\ApiAuth;
+
+   // ...
+   $initAuth  = new ApiAuth();
+   $auth      = $initAuth->newAuth($settings);
+   $apiUrl    = "https://your-mautic.com";
+   $api       = new MauticApi();
+   $reportApi = $api->newApi("reports", $auth, $apiUrl);
+
+.. vale off
+
+Get report
+**********
+
+.. vale on
+
+.. code-block:: php
+
+   <?php
+
+   //...
+   // Simple get all with default options:
+   $report = $reportApi->get($id);
+
+   // Or define exactly what rows you want:
+   $limit    = 100;
+   $page     = 2;
+   $dateFrom = \DateTime('1 week ago');
+   $dateTo   = \DateTime('now');
+   $report   = $reportApi->get($id, $limit, $page, $dateFrom, $dateTo);
+
+Get an individual Report by ID.
+
+.. vale off
+
+**HTTP Request**
+
+.. vale on
+
+``GET /reports/ID``
+
+Or define query parameters like this:
+
+``GET /reports/3?dateFrom=2017-01-01&dateTo=2018-01-01&limit=5&page=3``
+
+**Response**
+
+``Expected Response Code: 200``
+
+.. code-block:: json
+
+   {
+     "totalResults": 3990,
+     "data": [
+       {
+         "id2": "12",
+         "email1": "john@doe.email",
```

Comment:
```suggestion
         "email1": "john@example.com",
```

Reply:
Let's be more inclusive with the name examples we use. I selected some from https://developers.google.com/style/examples#example-person-names